src/main/java/us/kilroyrobotics/Robot.java

```diff
@@ -100,9 +104,11 @@ public void autonomousInit() {
                 Commands.sequence(autoDelayCommand, m_robotContainer.getAutonomousCommand());
 
         if (m_autonomousCommand != null) {
-            m_robotContainer.wristSetCoralStation.schedule();
+            m_robotContainer.wrist.setAngle(CoralMechanismConstants.kIntakingAngle);
```

Comment:
why are we doing this here instead of in the state machine?

Reply:
Testing something because state machine originally was setting the angle to `kStartingAngle` (which is just 0 as the physical initial position, but not where the code should be).

I'll remove it.